src/libopensc/card-cardos.c

```diff
@@ -886,6 +1078,7 @@ static int
 cardos_compute_signature(sc_card_t *card, const u8 *data, size_t datalen,
 			 u8 *out, size_t outlen)
 {
+	cardos_data_t* priv;
```

Comment:
clang-format suggested the following change:


```
-	cardos_data_t* priv;
+	cardos_data_t *priv;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->